src/components/ChangePlan/Card/Card.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+
+const Card = ({ data }) => {
+
+  return (
+    <div class="dp-card">
+      <h3>{data.title}</h3>
+      <p>{data.description}</p>
+      <p>{data.descriptionPlan}</p>
+      <div class="dp-price" data-start={data.price.initialText}>
```

Comment:
Why do you use data for rendering information?

Probably a _span_ is better than this. Did you verify how other people implement micro-formats? (the same for other content)

Reply:
i used the layout implemented by @GusBaamonde , we can change to `span` ?

Reply:
Seeing it now they are right, we can eliminate the data of "dp-price" and put a "span", "dp-money" we must leave it because it is a dynamic data. Webapp uses only u $ d, but sites use Mexican pesos, Colombian pesos, dollars, etc.

Reply:
I change it now in the layout so I arrange the elements well. Bad mine, sorry!